packages/plugin-ext/src/main/browser/dialogs-main.ts

```diff
@@ -161,4 +164,21 @@ export class DialogsMainImpl implements DialogsMain {
         return undefined;
     }
 
+    async $showUploadDialog(options: UploadDialogOptionsMain): Promise<string[] | undefined> {
+        const rootStat = await this.getRootUri(options.defaultUri);
+
+        // Fail if root not fount
+        if (!rootStat) {
+            throw new Error('Unable to find the rootStat');
```

Comment:
In case it's supposed to be shown to the user, can we provide some more meaningful message for him/her?

Reply:
Message has been updated